packages/react/src/components/Tabs/TabPanel.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+interface TabPanelProps {
+  index: number;
+  value: number;
+  children?: React.ReactNode;
+  className?: string;
+}
+
+const TabPanel = (props: TabPanelProps) => {
+  const { children, index, value, className, ...other } = props;
+
+  return (
+    <div
+      hidden={value !== index}
+      id={`tabpanel-${index}`}
```

Comment:
This is a bit problematic, [`aria-controls` is specified by the tab](https://github.com/dequelabs/cauldron/blob/e65928306ad11abec9b2cb319b8cfd0ba5cdb4aa/packages/react/src/components/Tabs/Tab.tsx#L32) but is potentially pointing to an id that is not unique because you could have multiple tab panels on the same page which would lead to duplicate ids.

Ideally we need some way to uniquely identify a set of tabs and their panels while avoiding duplicate ids.

`aria-labelledby` has the same issue directly below this.

There's a couple of different ways we could solve this, we could uniquely generate an id for each `<Tabs>` component that gets created and pass that identifier down to each tab/panel to control it.

```jsx
<Tabs>
  <Tab>1</Tab>
  <Tab>2</Tab>
  <Tab>3</Tab>
  <TabPanel>Panel 1</TabPanel>
  <TabPanel>Panel 1</TabPanel>
  <TabPanel>Panel 1</TabPanel>
</Tabs>
```

This allows for an accessible by default behavior. If the consumer cares about the specific ids, they should still be allowed to override the autogenerated default.

```jsx
<Tabs>
  <Tab id="tab-1" aria-controls="panel-1">1</Tab>
  <Tab id="tab-2" aria-controls="panel-1">2</Tab>
  <Tab id="tab-3" aria-controls="panel-1">3</Tab>
  <TabPanel id="panel-1" aria-labelledby="tab-1">Panel 1</TabPanel>
  <TabPanel id="panel-2" aria-labelledby="tab-1">Panel 1</TabPanel>
  <TabPanel id="panel-3" aria-labelledby="tab-1">Panel 1</TabPanel>
</Tabs>
```